mangaki/settings.template.ini

```diff
@@ -1,47 +1,51 @@
-[debug]
-  DEBUG = True
-
-[email]
-  EMAIL_BACKEND = django.core.mail.backends.console.EmailBackend
-
-[secrets]
-# Choose your own password or generate it with `pwgen -s -c 60 1`
-  SECRET_KEY = CHANGE_ME
-  DB_PASSWORD = YOUR_POSTGRE_PASSWORD
-
-#[deployment]
-#  MEDIA_ROOT = <base directory for media files>
-#  STATIC_ROOT = <base directory for static files
-
-#[hosts]
-#  ALLOWED_HOSTS = <see https://docs.djangoproject.com/fr/1.10/ref/settings/#allowed-hosts> 
-
-#[mal] # Used to get posters and user lists
-#  MAL_USER = 
-#  MAL_USER_AGENT = 
-
-#[anidb]
-#  ANIDB_CLIENT = 
-#  ANIDB_VERSION = 1
-
-#[pgsql]
-#  DB_HOST = <defaults to 127.0.0.1>
-#  DB_NAME = <defaults to mangaki>
-#  DB_USER = <defaults to django>
-
-#[sentry] (not required, only if you want to enable Sentry support)
-#  DSN = <sentry DSN>
-
-#[secrets]
-#  MAL_PASS = 
-
-#[smtp]
-#  EMAIL_HOST = 
-#  EMAIL_HOST_PASSWORD = 
-#  EMAIL_HOST_USER = 
-#  EMAIL_PORT = 
-#  EMAIL_SSL_CERTFILE = 
-#  EMAIL_SSL_KEYFILE = 
-#  EMAIL_TIMEOUT = 
-#  EMAIL_USE_SSL = 
-#  EMAIL_USE_TLS = 
+[debug]

```

Comment:
Seems like you had a whitespace change in the whole file, do you know what happened?